examples/website/basic-basemap-mapbox/package.json

```
@@ -9,15 +9,15 @@
     "build": "vite build"
   },
   "dependencies": {
-    "deck.gl": "^8.9",
-    "hubble.gl": "^1.4.0",
+    "deck.gl": "^9.1",
+    "hubble.gl": "file:../../../modules/main",
+    "mapbox-gl": "^3.10.0",
+    "popmotion": "11.0.5",
```

```suggestion
    "popmotion": "9.3.1",
```

Unless popmotion's ESM module issues have been fixed, and they reverted some export changes, I think this still must be pinned to this exact version. Long term, I think it makes sense to fork what we use from popmotion instead of depending on an aging locked dep.